.github/workflows/ci.yaml

```diff
@@ -3,6 +3,7 @@ on:
   pull_request:
     branches:
       - main
+      - htf-migration
```

Comment:
Thoughts on loosening up the target branches for CI @ijpulidos and @mikemhenry ?

I suspect PRs into PRs is going to be quite common, so maybe just setting this to `*` would make sense?